### ISSUE TITLE
Weird on bitbucket

### ISSUE THREAD

**Issue by Epistol** [kefranabg/readme-md-generator#60]
It's all messed up until the description after the tags :/  
```
 <h1><p><h1 align="center">Welcome to XXX 👋</h1> <p> <a href="https://twitter.com/XXX"> <img alt="Twitter: XXX" src="https://img.shields.io/twitter/follow/XXX.svg?style=social" target="_blank" /> </a> </p>```

**Comment by apatrascu** [comment]
Unfortunately you cannot center text on bitbucket because the platform doesn't support arbitrary HTML in Markdown. See link : https://bitbucket.org/tutorials/markdowndemo

**Comment by Epistol** [comment]
Unfortunately, that's the basic readme generator way of handling the file ^^'

**Comment by kefranabg** [comment]
I just created a good first issue to fix this problem 👉 #74

**Comment by kefranabg** [comment]
Since `v0.5.0` you can choose to generate template without HTML. See [changelog](https://github.com/kefranabg/readme-md-generator/blob/master/CHANGELOG.md).
